cmd/apikey_save.go

```diff
@@ -1,26 +1,51 @@
 package cmd
 
 import (
+	"bufio"
 	"fmt"
 	"os"
 
 	"github.com/civo/civogo"
 	"github.com/civo/cli/config"
 	"github.com/civo/cli/utility"
 	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
 var apikeySaveCmd = &cobra.Command{
 	Use:     "save",
 	Aliases: []string{"add", "store", "create", "new"},
 	Short:   "Save a new API key",
-	Args:    cobra.MinimumNArgs(2),
+	Args:    cobra.MinimumNArgs(0),
 	Example: "civo apikey save NAME KEY",
 	Run: func(cmd *cobra.Command, args []string) {
-		config.Current.APIKeys[args[0]] = args[1]
 
+		var username, apikey string
+		var err error
+		if len(args) == 0 {
+			reader := bufio.NewReader(os.Stdin)
+			fmt.Printf("Enter Username: ")
```

Comment:
This should be "Enter a nice name for this account/API Key:" - it's not their username.